src/util/gitinfostore.cpp

```diff
@@ -0,0 +1,37 @@
+#include "util/gitinfostore.h"
+
+#define GIT_INFO
+#include "gitinfo.h"
+
+//static
+const char* GitInfoStore::branch() {
+    return GIT_BRANCH;
+};
+
+//static
+const char* GitInfoStore::describe() {
+    return GIT_DESCRIBE;
+};
+
+//static
+const char* GitInfoStore::date() {
+    return GIT_COMMIT_DATE;
+};
+
+//static
+int GitInfoStore::commitCount() {
+#ifdef GIT_COMMIT_COUNT
+    return GIT_COMMIT_COUNT;
+#else
+    return 0;
```

Comment:
-1 maybe? or use std::optional?

Reply:
0 fits, because we have 0 commits if we are off the git worktree